R-package/configure.ac

```diff
@@ -0,0 +1,167 @@
+### configure.ac                    -*- Autoconf -*-
+# Template used by Autoconf to generate 'configure' script. For more see:
+#   * https://unconj.ca/blog/an-autoconf-primer-for-r-package-authors.html
+#   * https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Configure-and-cleanup
+
+AC_PREREQ(2.69)
+AC_INIT([lightgbm], [2.3.2], [], [lightgbm], [])
```

Comment:
One another approach: https://github.com/cran/HDclust/blob/571f27be98eaffafca0d9b3fe2e64ce2ccaead68/configure.ac#L10

Reply:
oooooooooh ok! I think you might be better at Googling than me, because I definitely searched for how to do this!

Yes I think reading it from a file is a lot better. I'll do that here